### PR TITLE
add some docs for 0.9.0

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: yuxing.lyx <yuxing.lyx@alibaba-inc.com>

### CODE REVIEW

File: src/docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -1,72 +1,92 @@
 # Quick start
 
+This guide covers how you can quickly get started using Sealer.
+
+## PREREQUISITES
```
Comment (kakaZhou719) [inline src/docs/getting-started/quick-start.md:5]: Prerequisite?

File: src/docs/getting-started/quick-start.md [hunk 2]
```diff
@@ -1,72 +1,92 @@
...
+1. Installing the sealer locally.
```
Comment (kakaZhou719) [inline src/docs/getting-started/quick-start.md:6]: Install

File: src/docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -1,72 +1,92 @@
...
+2. Prepare six hosts that meet the following criteria, and record their IPs and passwo
```
Comment (kakaZhou719) [inline src/docs/getting-started/quick-start.md:7]: passwo: password

File: src/docs/getting-started/quick-start.md [hunk 4]
```diff
@@ -1,72 +1,92 @@
...
+   1. Hardware: 4C8G + 50G(System Disk)
+   2. OS: CentOS/RHEL 7.5、CentOS/RHEL 7.6、CentOS/RHEL 7.7、CentOS/RHEL 7.8、CentOS/RHEL 7.9
+   3. Kernel: 4.18.*（推荐）、4.19.* 、 3.10.*（>=3.10.0-1160）
```
Comment (kakaZhou719) [inline src/docs/getting-started/quick-start.md:10]: （推荐) (recommend)

File: src/docs/getting-started/quick-start.md (hunk 3 above)
Comment (kakaZhou719) [inline src/docs/getting-started/quick-start.md:7]: this is not prerequisite

File: src/docs/getting-started/quick-start.md [hunk 5]
```diff
@@ -1,72 +1,92 @@
...
+```shell
+# you can push the app image to docker hub, Ali ACR, or Harbor
+sealer tag sealer-io/nginx:latest {registryDomain}/sealer-io/nginx:latest
+sealer push {registryDomain}/sealer-io/nginx:latest
 ```
 
-## Push the CloudImage to the registry
+## Clean the cluster
+
+Some information of the basic settings will be written to the Clusterfile and stored in /root/.sealer/Clusterfile.
```
Comment (kakaZhou719) [inline src/docs/getting-started/quick-start.md:88]: Some information of the basic settings will be written to the cluster and stored in /root/.sealer/Clusterfile locally.

**Comment by kakaZhou719** [comment]
do we need save pictures in sealer repo : [github](https://github.com/sealerio/sealer/discussions/1939)

File: src/docs/getting-started/quick-start.md (hunk 1 above)
Comment (Starnop) [reply src/docs/getting-started/quick-start.md:5]: use plurality here

File: src/docs/getting-started/quick-start.md (hunk 3 above)
Comment (Starnop) [reply src/docs/getting-started/quick-start.md]: What does that mean?

**Comment by Starnop** [comment]
> do we need save pictures in sealer repo : [sealer repo](https://github.com/sealerio/sealer/discussions/1939)

This is one of the ways to store images on github

File: src/docs/getting-started/quick-start.md (hunk 1 above)
Comment (kakaZhou719) [reply src/docs/getting-started/quick-start.md:5]: i mean ,do we need lower case for the titles word?

File: src/docs/getting-started/quick-start.md (hunk 3 above)
Comment (kakaZhou719) [reply src/docs/getting-started/quick-start.md]: do we need six hosts  for the prerequisite?

**Pull request merged**